tgbot/config.py

```diff
@@ -167,27 +135,20 @@ class Config:
     """
 
     tg_bot: TgBot
-    misc: Miscellaneous
-    db: Optional[DbConfig] = None
-    redis: Optional[RedisConfig] = None
-
+    db: DbConfig
+    redis: RedisConfig
+    misc: Optional[Miscellaneous] = None
 
-def load_config(path: str = None) -> Config:
-    """
-    This function takes an optional file path as input and returns a Config object.
-    :param path: The path of env file from where to load the configuration variables.
-    It reads environment variables from a .env file if provided, else from the process environment.
-    :return: Config object with attributes set as per environment variables.
-    """
 
-    # Create an Env object.
-    # The Env object will be used to read environment variables.
-    env = Env()
-    env.read_env(path)
+def load_config() -> Config:
```

Comment:
Make sure we are able to pass the custom env file, for purposes of separation prod and dev environments